sdk/scheduler/src/main/java/com/mesosphere/sdk/specification/DefaultNetworkSpec.java

```diff
@@ -89,14 +77,18 @@ public int hashCode() {
         return HashCodeBuilder.reflectionHashCode(this);
     }
 
+    @Override
+    public String toString() {
```

Comment:
Might as well capture all the fields.

See other classes toString to see how we typically do it (Reflection)